### DISCUSSION TITLE
Entity ID  in events: state

### DISCUSSION THREAD

**Discussion by rosaldanha** [zachowj/node-red-contrib-home-assistant-websocket#595] [Q&A]
Hello, 
    I using this string in Entity field, using entity filter type Substring:

    binary_sensor.lk3c53boiler, binary_sensor.ds1e5dportaescritorio_2

![image](https://user-images.githubusercontent.com/576764/154655704-0f8e177b-bdca-448b-b1a6-deae0da0ad56.png)

   But i didn't get any output...

**Comment by zachowj** [comment]
One of the breaking changes for v0.42.0 was that substring no longer parses comma delimited strings into single entities. A type was added 'list'

https://github.com/zachowj/node-red-contrib-home-assistant-websocket/releases/tag/v0.42.0

**Reply by rosaldanha** [reply]
Thanks. I'll try to use the list type.
